lib/ansible/modules/packaging/os/rhn_register.py

```diff
@@ -286,7 +285,7 @@ def api(self, method, *args):
                 url = "https://%s/rpc/api" % self.hostname
             else:
                 url = "https://xmlrpc.%s/rpc/api" % self.hostname
-            self.server = xmlrpclib.Server(url, verbose=0)
+            self.server = xmlrpc_client(url)
```

Comment:
Oups, fixed in https://github.com/ansible/ansible/pull/26878/commits/3c2d147eee21f16abc7cfbe794c60ae154377d79.